internal/core/queries/get_reviews_by_id.go

```diff
@@ -53,6 +57,7 @@ func (qh GetReviewsByIDQueryHandler) Handle(ctx context.Context, query mediator.
 		Comments:           review.Comments,
 		Approved:           review.Approved,
 		ApprovedBy:         review.ApprovedBy,
+		Name:               fmt.Sprintf("%d %s %s", review.R.DeviceDefinition.Year, review.R.DeviceDefinition.R.DeviceMake.Name, review.R.DeviceDefinition.Model),
```

Comment:
refactor how name is built with above - since we do same thing in two spots

Reply:
I created an util function => BuildDeviceDefinitionName